test/db/cmd/cmd_ie

```diff
@@ -9,9 +9,7 @@ EXPECT=<<EOF
 
 0 entrypoints
 EOF
-EXPECT_ERR=<<EOF
-WARN: run r2 with -e bin.cache=true to fix relocations in disassembly
-EOF
+EXPECT_ERR=
```

Comment:
omg this is making relocs on by default. which is something we dont want yet. because im aware of many bugs still in the relocs parsing/patching, and the iocache layers is good for debugging that.

but still i think bin.cache was just a hack and we need `bin.applyrelocs` or sthg?

im happy to merge your pr if the current behaviour is not modified (choose overlay vs iocache) and then we can remove the bin.cache and call it bin.patchrelocs or wathever makes sense,

Reply:
I just removed the warnings, because they are annoying

Reply:
if your concern is debugging, i can give you some #!c scripts for that